1904-11-14.xml

```diff
@@ -0,0 +1,2181 @@
+                    at the French School of Law at Cairo is fixed for Thursday next, 17th inst., at
+                    3.30 p.m.</p>
+                <p>MEDICAL LICENSES.-Drs. J. S. Bordach and Emin Malouf have been authorised by the
+                    Ministry of the Interior to practise in Egypt. M. Franz Moldehuke has been
+                    authorised to practise as a pharmacist.</p>
+                <p>BRITISH SOLDIER DROWNED. On Saturday afternoon the corpse of Private Sinclair, of
+                    the 2nd Battalion Royal Inniskilling Fusiliers, was found in the Nile at Ghizeh.
+                    The funeral took place yesterday afternoon.</p>
+                <p>NEWSPAPER LIBEL CASE-Mr. Gammal, a wealthy Alexandrian, has brought an action for
+                    libel against the proprietor and editor of the Tamiel, a local Arabic newspaper.
+                    £1000 damages are claimed.</p>
+                <p>THE ANNIVERSARY of the death of Ismail Pasha was commemorated yesterday evening
+                    at the Daira of Kasr el-Aly by a religious cere- mony. Some verses of the Koran
+                    were read and'alms were given to the poor at the ceme- tery of the Khedivial
+                    family.</p>
+                <p>Of these, 10 The transit</p>
+                <p>SUEZ CANAL. On the 10th inst. 11 vessels passed through the Suez Canal. were
+                    British and 1 was Prench. and passenger dues for the day amounted to 314,821.49
+                    frs. The total transit and passenger dues collected from the 1st inst. to that
+                    date on 120 vessels amounted to 3,174,893.54 frs.</p>
+                <p>MR. JOHN B. CAPPARI wishes to inform his clients that, owing to the expiration of
+                    the lease of his old premises in the Grand Square his offices and stores have
+                    been removed to 8, Rue de l'Eglise Debbane and 6, Boulevard de Ramleh. As the
+                    fittings of the offices, shop, and stores are not yet completed, some of the
+                    back stores will be used temporarily, with entrance from 8, Rue de l'Eglise
+                    Debbane.</p>
+            </div>
+            <cb n="2">
+                
+            </cb>
+            
+            <pb n="4"/>
+            <div type="page" n="4"
+                facs="https://raw.githubusercontent.com/dig-eg-gaz/page-images/master/YYYY-page-images-N/YYYY-MM-DD-p4.jpg"> </div>
+            <pb n="5"/>
+            <div type="page" n="5"
+                facs="https://raw.githubusercontent.com/dig-eg-gaz/page-images/master/YYYY-page-images-N/YYYY-MM-DD-p5.jpg"> </div>
+            <pb n="6"/>
+            <div type="page" n="6"
+                facs="https://raw.githubusercontent.com/dig-eg-gaz/page-images/master/YYYY-page-images-N/YYYY-MM-DD-p6.jpg"> </div>
+
+        </body>
+        <note>
```

Comment:
This is the rest of page 3 OCR, which somehow wound up in a note. It happens, no big deal. If you drag the whole note into the page 3 div, then delete the `<note>` tag and it should be fine.